sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderAsyncClient.java

```diff
@@ -248,8 +261,13 @@
      * @return copyright attribution for the requested section of a tileset.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<MapAttribution> getMapAttribution(TilesetID tilesetId, int zoom, List<Double> bounds) {
-        return this.serviceClient.getMapAttributionAsync(tilesetId, zoom, bounds);
+    public Mono<Response<MapAttribution>> getMapAttributionWithResponse(TilesetID tilesetId, int zoom, List<Double> bounds) {
```

Comment:
bounds is one of the properties you were supposed to optimize with a BoundingBox